src/App.tsx

```diff
@@ -2,15 +2,23 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const addMovie = (movie: Movie) => {
+    setMovies(movies.concat(movie));
```

Comment:
when updating the state based on the previous state’s value always use function that receives the previous state as its argument
```suggestion
    setMovies(prevMovies => prevMovies.concat(movie));
```